src/components/animations/ButtonPressAnimation/ButtonPressAnimation.android.js

```diff
@@ -297,9 +301,18 @@ export default function ButtonPressAnimation({
       transformOrigin={normalizedTransformOrigin}
       wrapperStyle={wrapperStyle}
     >
-      <Content pointerEvents="box-only" style={style}>
+      <View pointerEvents="box-only" style={[cx.overflow, style]}>
         {children}
-      </Content>
+      </View>
     </ButtonElement>
   );
 }
+
+const cx = StyleSheet.create({
```

Comment:
Could you use a different name than `cx`? I have a hard time knowing what is it about. I think I already changed something about the `sz` in animated styles in another component

Reply:
cx is actually a really popular name for such things if you've ever used css-modules in react.

```js
import classnames from 'classnames/bind'
import s from './styles.module.css'

const cx = classnames.bind(s);
```

but yeah I'll consider something else.

Reply:
I think cs is ok